src/Farfetch.LoadShedding.Prometheus/Metrics/HttpRequestsConcurrencyItemsGauge.cs

```diff
@@ -22,16 +21,27 @@ public class HttpRequestsConcurrencyItemsGauge : MetricBase<Gauge>
         protected override string DefaultName => "http_requests_concurrency_items_total";
 
         /// <summary>
-        /// Sets the value of the gauge.
+        /// Increments the value of the gauge.
+        /// </summary>
+        /// <param name="method">The method.</param>
+        /// <param name="priority">The priority.</param>
+        public void Increment(string method, string priority)
+        {
+            this.Metric?
+                .WithLabels(method, priority)
```

Comment:
**question for maintainers**, I didn't see any use of the method in this framework or even in the Grafana dashboard.

What are the plans for this type of custom labels?

Reply:
Yes, I would say that the http method is useless for now and maybe we can remove that instead of creating a coupling.